src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import {
+  createStore, combineReducers, applyMiddleware,
+} from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import leaderboardReducer from './leaderboard/leaderboardRedux';
+
+const reducer = combineReducers({
+  leaderboardReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger, thunk),
+);
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data and errors displayed on the browser console.